### DISCUSSION TITLE
error: #error "you should set MAGICKCORE_HDRI_ENABLE"

### DISCUSSION THREAD

**Discussion by aohanhongzhi** [ImageMagick/ImageMagick#6580] [Help]
### ImageMagick version

7

### Operating system, version and so on

ubuntu 20.04

### Description

how to set MAGICKCORE_HDRI_ENABLE?

![image](https://github.com/ImageMagick/ImageMagick/assets/37319319/ec9bc04d-b921-479c-91a5-754f90fbf292)

**Comment by dlemstra** [comment]
Which exact version of ImageMagick are you using?

**Comment by urban-warrior** [comment]
Try this command:

```
pkg-config --cflags --libs MagickCore
```

Notice it sets the requisite MAGICKCORE_HDRI_ENABLE and MAGICKCORE_QUANTUM_DEPTH settings:

```
-I/usr/include/ImageMagick-7 -fopenmp -DMAGICKCORE_HDRI_ENABLE=1 -DMAGICKCORE_QUANTUM_DEPTH=16 -lMagickCore-7.Q16HDRI
```

See https://imagemagick.org/script/magick-core.php.
